solr/core/src/java/org/apache/solr/search/DocSetQuery.java

```diff
@@ -31,9 +31,10 @@
 import java.util.Objects;
 
 /**
- * A class that accesses Queries based on a DocSet
+ * A Query based on a {@link DocSet}.  The un-boosted score is always 1.
  *
- * Refer SOLR-15257
+ * @see DocSet#makeQuery()
+ * @since 9.0
  */
 public class DocSetQuery extends Query implements DocSetProducer{
```

Comment:
Would this class make sense as not public?

Reply:
I agree; I've been thinking the same thing